app/gui/src/dashboard/layouts/Settings/lambdaKinds.ts

```diff
@@ -0,0 +1,274 @@
+/** @file Kinds of lambda. */
+import type { IconPropSvgUse, SvgUseIcon } from '#/components/AriaComponents'
```

Comment:
I'm okay to leave it as a temporary solution. But this _must_ be revisited someday.